contrib/opencensus-ext-stackdriver/setup.py

```diff
@@ -40,11 +40,12 @@
     long_description=open('README.rst').read(),
     install_requires=[
         'google-cloud-trace >= 0.20.0, < 1.0.0',
-        'opencensus >= 0.3.0, < 1.0.0',
+        'google-cloud-monitoring >= 0.31.0, < 1.0.0',
+        'opencensus >= 0.3.0, < 0.4.0',
```

Comment:
Note that we need `<0.4.0` here since there were breaking package changes (`exporters.base` to `base_exporter`). We should restrict contrib package dependencies to a specific minor version on each release.